### PR TITLE
Updating to new build system

### PR DESCRIPTION
**Pull request by mpodwysocki**
Things to consider before you submit the PR:

* [ ] Are tests passing locally?
* [ ] Are the files formatted correctly?
* [ ] Did you add unit tests?
* [ ] Did you test your change with either the sample apps that are included in the repository or with a blank app that uses your change?

## Description

A few sentences describing the overall goals of the pull request.

## Related PRs or issues

List related PRs and other issues.

## Misc

Add what's missing, notes on what you tested, additional thoughts or questions.

### CODE REVIEW

File: Scripts/framework-version.sh [hunk 1]
```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 
 # Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License.
```
Comment (MatkovIvan) [inline Scripts/framework-version.sh:4]: Is this change expected?

File: Scripts/sort-schemes.sh [hunk 2]
```diff
@@ -0,0 +1,118 @@
+#!/bin/bash
```
Comment (MatkovIvan) [inline Scripts/sort-schemes.sh:1]: 99% sure that you do not need this script

Comment (mpodwysocki) [reply Scripts/sort-schemes.sh:1]: Removing

File: Scripts/framework-version.sh (hunk 1 above)
Comment (mpodwysocki) [reply Scripts/framework-version.sh:4]: Oh woops, yeah, need to remove that

**Pull request merged**